PROJECT-BRIEF.md

```diff
@@ -1,59 +1,186 @@
+| {DEV_2} & {DEV_3}     | [10. As a user, I want to be alerted when I’m entering an empty item or an item that's already on my list.]({REPO_URL}/issues/10)                                                                       |
+| {DEV_1} & {DEV_4}     | [11. As an item, I want my estimated next purchase date to be computed at the time my purchase is recorded in the database so the app can learn how often I buy different items.]({REPO_URL}/issues/11) |
+
+| 📖 Learning modules                                                        | Description |
+| -------------------------------------------------------------------------- | ----------- |
+| [Navigating finances during a career change](https://youtu.be/xtkgOmFPUJU) |             |
+
+| 👋 Sync meetings | Agenda                                                                                                                                                            |
+| ---------------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| Weekly Sync      | - Demo <br> - Retro <br> - Next week's issues                                                                                                                     |
+| Pair Programming | Schedule a pairing session(s) with your partner early in the week to complete the development work required to meet the acceptance criteria of your Github issue. |
+| Optional         | Office Hours                                                                                                                                                      |
+
+### Week 6, {DATES_WEEK_6}
+
+| 🤝 Developer pairings | 🖥 Assigned Github issue                                                                                                                                                                              |
+| --------------------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| {DEV_3} & {DEV_1}     | [12. As a user, I want to be able to delete items from my shopping list so that my list isn’t cluttered with items I don’t want to buy in the future.]({REPO_URL}/issues/12)                          |
+| {DEV_4} & {DEV_2}     | [13. As a user, I want to view a list of my shopping list items in order of how soon I am likely to need to buy each of them again so that it’s clear what I need to buy soon.]({REPO_URL}/issues/13) |
+
+| 📖 Learning modules                                                                  | Description |
+| ------------------------------------------------------------------------------------ | ----------- |
+| [What to ask your interviewers](./learning-modules/what-to-ask-your-interviewers.md) |             |
```

Comment:
The transcriptions from the learning modules throw a 404 error.

Reply:
Possible Description: In this module, you'll learn how to ask questions during interviews to ensure the position/company is a good fit for both you and your potential employer. 

_Bulleted list, if needed:_
You'll learn how to:

- reflect on previous workplace experiences
- create questions based on your reflections